.github/workflows/nightly_vtk_master.yml

```diff
@@ -0,0 +1,223 @@
+name: Nightly VTK master
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+
+#----------------------------------------------------------------------------
+# Check nightly: Checkout VTK and check if the last merged commit is less than 24 hours old
+#----------------------------------------------------------------------------
+  check_nightly:
+    runs-on: ubuntu-latest
+    name: Check nightly is needed
```

Comment:
```suggestion
    name: Check nightly if needed
```

Reply:
hum no, this action "check if the nightly is needed". I'll just rename it "Check nightly".